### DISCUSSION TITLE
:disabled prop for NuxtLink

### DISCUSSION THREAD

**Discussion by QaahirStewart** [nuxt/nuxt#16187] [General]
If using NuxtLink styled as a button. How would we  :disabled. If we can't can we add this feature?

**Comment by IsraelOrtuno** [comment]
Tricky thing. Based on https://www.scottohara.me/blog/2021/05/28/disabled-links.html what you could do is just to remove the `to` and play with `aria-disabled` and `role` as links don't have a `disabled` attribute.
